UltiSnips/django.snippets

```diff
@@ -1,238 +1,367 @@
 priority -50
 
-# Generic Tags
-snippet %
-{% ${1} %}${2}
+# This files will define django snippets from sublime text djaneiro
+
+###########################################################################
+#                             FORMS SNIPPETS                              #
+###########################################################################
+snippet form "Form"
+class ${1:FORMNAME}(forms.Form):
+
+	${2:# TODO: Define form fields here}
+endsnippet
+
+snippet modelform "ModelForm"
+class ${1:MODELNAME}Form(forms.ModelForm):
+
+	class Meta:
+		model = $1
+endsnippet
+
+snippet fbool "BooleanField (fbool)"
```

Comment:
We decided to remove () in description fields, i.e. do not repeat the trigger please.